extensions/notebook/src/book/bookModel.ts

```diff
@@ -334,6 +334,10 @@ export class BookModel {
 		return treeItems;
 	}
 
+	private setHierarchyId(iteration: number, hierarchyId?: string): string {
+		return hierarchyId ? path.join(hierarchyId, iteration.toString()) : iteration.toString();
```

Comment:
Seems a bit odd to use path.join for this since this isn't a file system path. Any reason not to just use / all the time?